wire-ios/Wire-iOS Tests/SwitchBackendConfirmationViewTests.swift

```diff
@@ -0,0 +1,52 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import XCTest
+@testable import Wire
+
+final class SwitchBackendConfirmationViewTests: XCTestCase {
+
+    let helper = SnapshotHelper()
+        .withLayout(.device(config: .iPhone13))
```

Comment:
Is `iPhone 13` used in other places too? Maybe we can have a new branch to update to newer iPhones then

Reply:
It was the highest version, I think because the snapshot library is a bit old. I would like to have simply passed the view without specifying the layout but I think because it contains a scroll view it needs an explicit frame.

Reply:
Even in the latest version of the snapshot framework iPhone13 is the newest model.